### PR TITLE
PR for #56 - Updated NuGet packages

### PR DESCRIPTION
**Pull request by jamesmcroft**
This PR includes a change to the XPlat referenced NuGet packages to update them to the latest supported versions. 

Updated packages including:
- Microsoft.NETCore.UniversalWindowsPlatform to 6.1.9
- MADE to 0.2.19058.1
- CommonServiceLocator to 2.0.4
- MvvmLightLibs to 5.4.1.1
- Newtonsoft.Json to 12.0.1
- MS Test to 1.4.0
- Xamarin.Support to 28.0.0.1

### CODE REVIEW

**Comment by jamesmcroft** [comment]
@tom-made can you take a look at this please?

File: XPlat.ApplicationModel/XPlat.ApplicationModel.csproj [hunk 1]
```diff
@@ -31,7 +31,7 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'uap10.0' ">
-    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.1.7" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.1.9" PrivateAssets="All" />
```
Comment (tom-made) [inline XPlat.ApplicationModel/XPlat.ApplicationModel.csproj]: Is there a reason that we aren't using the 6.2.3 update like MADE?

Comment (jamesmcroft) [reply XPlat.ApplicationModel/XPlat.ApplicationModel.csproj]: 6.2.3 isn't available anymore and was pulled recently I believe. 6.1.9 is the latest although this shouldn't matter, it's a dev dependency.

Comment (tom-made) [reply XPlat.ApplicationModel/XPlat.ApplicationModel.csproj]: Cool no worries, will we need to make this change on MADE?

Comment (jamesmcroft) [reply XPlat.ApplicationModel/XPlat.ApplicationModel.csproj]: Possibly. I can take a look at that.

**Pull request merged**